uploadToLoki.js

```diff
@@ -144,5 +94,66 @@ const deployApp = async () => {
   await clearFiles();
   pushToLoki();
 };
-
+/**
+ * @param {import('./package.json')} p The "package.json" file for the application
+ */
+function makePageDataObject(p) {
+  const {
+    appInfo: { loki },
+  } = p;
```

Comment:
Where possible, try to avoid abbreviations in variable names and function names. It makes things more verbose, but I err on the side of verbosity vs nomenclature that's easy to lose track of.

Reply:
Done

Reply:
Renamed and added types